msal_extensions/windows.py

```diff
@@ -39,6 +39,15 @@ def raw(self):
         _MEMCPY(blob_buffer, pb_data, cb_data)
         return blob_buffer.raw
 
+_err_description = {
+    # Keys came from real world observation, values came from winerror.h (https://errors)
```

Comment:
> https://errors

The URL is incorrect.

Reply:
The "httpS" one was indeed a typo. I meant to use the "http" one.

And yes, I know it is a Microsoft internal website. Do you know some public website that provide similar information that we can refer to? If not, the `http://errors` is better than nothing. It at least leaves a trace for us to triage customer's reports.